main.js

```diff
@@ -100,6 +110,9 @@ Device.prototype = {
  * Chrome on Android, for example.
  */
 
+Devices.on("fastboot-start-polling", fastboot.startPolling.bind(fastboot));
+Devices.on("fastboot-stop-polling", fastboot.stopPolling.bind(fastboot));
```

Comment:
That would be great if you could require fastboot lazily from the event listener so that we load the module (and its deps) only when used.